c2rust-transpile/src/translator/mod.rs

```diff
@@ -4275,6 +4284,13 @@ impl<'c> Translation<'c> {
                     let expr =
                         expr.ok_or_else(|| format_err!("Casts to enums require a C ExprId"))?;
                     Ok(self.enum_cast(ty.ctype, enum_decl_id, expr, val, source_ty, target_ty))
+                } else if target_ty_ctype.is_floating_type() && source_ty_kind.is_bool() {
```

Comment:
This seems unrelated to `Debug`. Unrelated changes usually get separate PRs, but we can make an exception this time.

Reply:
Never mind, I just saw PR #1030 .